composer.json

```diff
@@ -86,6 +86,7 @@
         "google/cloud-spanner": "1.6.0",
         "google/cloud-speech": "0.15.0",
         "google/cloud-storage": "1.5.2",
+        "google/cloud-text-to-speech": "0.0.1",
```

Comment:
Small nit: I think we want this to just be `master`. It will make it easier when we go to use the automated tooling to bump the version at release time (not a huge deal, we can just manually fix if need be).